{
  super("Uncaught exception by thread: " + e.thread,e.exception);
}
